schema/divisions/defs.yaml

```diff
@@ -0,0 +1,142 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+description: Common schema definitions for divisions theme
+"$defs":
+  typeDefinitions:
```

Comment:
So far we had propertyDefinitions. Any reason why this is now type definitions?

Reply:
Yeah, I think `propertyDefinitions` is wrong based on how we're using it. (And I think I'm probably the one who created `propertyDefinitions`, so my mistake.)

Basically in the top-level `defs.yaml`, mostly what we're defining is reusable "types" that are then used as the schemas for properties. But we're not defining the properties themselves, just the types/schemas. So I think it would be more correct if we migrate to the `typeDefinitions` terminology.

Reply:
I like the `propertyDefinitions` name since it is just there to define possible properties that can be used by different features. In any case, I would like to have single value everywhere, whether it is `type` or `property` definitions. Having two different names doing the same thing seems messy.

Reply:
My point is that they aren't *properties*, they are types/schemas that may be used *for* properties, which is different.

Agreed it seems messy. I intend to refactor the other parts of the schema to align, but didn't want to change it all at once.